src/javascript/app/pages/user/account/settings/financial_assessment.js

```diff
@@ -45,10 +45,8 @@ const FinancialAssessment = (() => {
             });
         }
 
-        const is_professional = State.getResponse('get_account_status.professional') || State.getResponse('get_account_status.professional_requested');
-        if (is_professional) {
-            $('#trading_experience_form').setVisibility(0);
-        }
+        const is_professional = /professional|professional_requested/.test((State.getResponse('get_account_status') || {}).status);
```

Comment:
it would have to check exactly for professional term, right now it will catch anything that starts with professional. it needs to be `/^(professional|professional_requested)$/`

Reply:
you can do `State.getResponse('get_account_status.status')` instead

Reply:
should we check `professional|professional_requested` from `status` array?

Reply:
I think so, yes